repository/src/main/java/org/pentaho/platform/repository2/unified/jcr/RepositoryFileProxy.java

```diff
@@ -226,6 +232,8 @@ public String getCreatorId() {
           creatorId = (String) metadata.get( PentahoJcrConstants.PHO_CONTENTCREATOR );
         }
       }
+    } catch ( InvalidItemStateException | ItemNotFoundException | PathNotFoundException e ) {
+      getLogger().warn( "InvalidItemStateException. Probable cause: File does not exist anymore" );
```

Comment:
Only comment here - your log messages are all the same, and don't re-throw the exception. Understand not necessarily wanting to throw the exception, but there's no way to tell if the exception happened in getCreatorId() or execute() or anywhere else. Strongly suggest that the messages at least be differentiated so that the debugger can tell which method caught the exception. Something like: InvalidItemStateException in getCreatorId. Probable cause: File does not exist anymore.

Reply:
Totally agree with you...Will work on it